crates/rattler/src/install/python.rs

```diff
@@ -71,11 +71,11 @@ impl PythonInfo {
     /// Constructs a shebang that will run the rest of the script as Python.
     pub fn shebang(&self, target_prefix: &str) -> String {
         let target_path = Path::new(target_prefix).join(self.path());
-        let target_path = target_path.as_os_str().to_string_lossy();
+        let target_path = target_path.as_os_str().to_string_lossy().replace('\\', "/");
```

Comment:
hmm, unsure if that's the right move here. Do we use shebangs only on Unix? 

If we use shebangs on Windows (e.g. with `cygwin`) we should use `cygpath` to convert the path to a unix style path (`/c/Windows/System Profile/ ...`)

Reply:
Im not sure it's actually used on Windows. I copied this code from mamba. The launcher reads the shebang to determine the python instance to start I think. I would have to dig deeper though..

Reply:
Actually, thinking about it. The `target_prefix` that is passed in should already be the correct prefix (cygwin or not). Since windows works both with `\` or `/` I think this is fine/